test/unit-test/project.yml

```diff
@@ -81,20 +81,20 @@
 
 :tools:
   :test_compiler:
-     :executable: clang
-     :name: 'clang compiler'
+     :executable: gcc
```

Comment:
change unit test compiler to gcc, since clang (x86 target) conflict with my clang arm target